README.md

```diff
@@ -319,77 +322,204 @@ spec:
+  client_secret: ********
+  instance_guid: your-sample-instance-guid // The service instance ID
+  instance_name: sample-instance // Taken from the service instance external_name field if set. Otherwise from metadata.name
+  plan: sample-plan // The service plan name                
+  type: sample-service  // The service offering name
 ```
 
 ###### Credentials as JSON Object
-To show credentials returned from the broker as a JSON object, use the 'secretKey' attribute in the service binding spec.
 
-The value of 'secretKey' is the name of the key that stores the credentials in JSON format.
+To show credentials returned from the broker within the `Secret` resource  as a JSON object, use the 'secretKey' attribute in the `ServiceBinding` spec.
+The value of this 'secretKey' must be the name of the key that stores the credentials in JSON format:
 
-```bash
-#Credentials
-your-secretKey-value:
-{
-  uri: https://my-service.authentication.eu10.hana.ondemand.com
-  username: admin
-  password: ********
-}
+Input:
 
-#Service Instance info
-instance_guid: <instance_guid> // The service instance ID
-instance_name: my-service-btp-name // Taken from the service instance external_name field if set. Otherwise from metadata.name 
-plan: sample-plan // The service plan name
-type: sample-service // The service offering name
+```yaml
+apiVersion: services.cloud.sap.com/v1
+kind: ServiceBinding
+metadata:
+  name: sample-binding
+spec:
+  serviceInstanceName: sample-instance
+  secretKey: your-secretKey-value
+```
+Output:
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: sample-binding
+stringData:
+    your-secretKey-value:
+    {
+      uri: https://my-service.authentication.eu10.hana.ondemand.com,
+      client_id: admin,
+      client_secret: ********
+    }
+    instance_guid: your-sample-instance-guid // The service instance ID
+    instance_name: sample-binding // Taken from the service instance external_name field if set. Otherwise from metadata.name 
+    plan: sample-plan // The service plan name
+    type: sample-service // The service offering name
 ```
 
 ###### Credentials and Service Info as One JSON Object
-To show both credentials returned from the broker and service instance info as a JSON object, use the 'secretRootKey' attribute in the service binding spec.
 
-The value of 'secretRootKey' is the name of the key that stores both credentials and serivce instance info in JSON format.
+To show both credentials returned from the broker and additional `ServiceInstance` attributes as a JSON object, use the 'secretRootKey' attribute in the `ServiceBinding` spec.
+
+The value of 'secretRootKey' is the name of the key that stores both credentials and `ServiceInstance` info in JSON format.
 
-```bash
-your-secretRootKey-value:
+Input:
+
+```yaml
+apiVersion: services.cloud.sap.com/v1
+kind: ServiceBinding
+metadata:
+  name: sample-binding
+spec:
+  serviceInstanceName: sample-instance
+  secretRootKey: your-secretRootKey-value
```

Comment:
use 'myBindingData'

Reply:
Wouldn't this be myCredentialsAndInstanceData if already?? @avilupu